src/frontend/src/utils/userIntent.ts

```diff
@@ -15,13 +15,25 @@ export const intentFromUrl = (url: URL): UserIntent => {
   }
 };
 
-export const verbFromIntent = (intent: UserIntent): string => {
+export const authenticateIntent = (intent: UserIntent): string => {
   switch (intent.kind) {
     case "addDevice":
-      return "add your new device to your Internet Identity";
+      return "Authenticate to add your new device";
     case "auth":
-      return "authenticate using your Internet Identity";
+      return "Authenticate using Internet Identity";
     case "manage":
-      return "manage your Internet Identity";
+      return "Authenticate using Internet Identity";
```

Comment:
Did something go wrong here? Should it say "Manage your Internet Identity"?

Reply:
This is the bit I talked about where previously it said "Authenticate to manage your Internet Identity". Now we want the "manage your Internet Identity" bit to disappear but just saying "Authenticate to <nothing>" doesn't work so I had to change things around a bit.

Reply:
Ok, thanks for explaining. I also added this point to [II-36](https://dfinity.atlassian.net/browse/II-36) in case we want to address this again when cleaning this file up.